src/app/libp2p_helper/src/bitswap_storage.go

```diff
@@ -67,21 +70,25 @@ func UnmarshalRootBlockStatus(r []byte) (res RootBlockStatus, err error) {
 	return
 }
 
-func (bs *BitswapStorageLmdb) ViewBlock(key [32]byte, callback func([]byte) error) error {
-	return bs.blockstore.View(BlockHashToCid(key), callback)
+func (bs *BitswapStorageLmdb) StoreBlocks(ctx context.Context, blocks []blocks.Block) error {
```

Comment:
Seems like this is the wrong PR for this change, but it's small so we can keep it here in interest of moving this forward.

Reply:
Well, actually it belongs to this PR as the change is needed only with the latest `go-bitswap` version.